CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 ## Unreleased Version
 * PayPalNativeCheckout
     * Update to version 0.109.0 that fixes shipping callback bug
+* Bump minimum supported deployment target to iOS 14+
```

Comment:
nit: looks like the [github actions release step](https://github.com/paypal/iOS-SDK/blob/main/.github/workflows/release.yml#L20) does a grep for `## unreleased` and the changelog currently uses `## Unreleased Version`. We should update that in this PR as well to ensure it's properly formatted for future releases.

Reply:
@jaxdesmarais good catch! Going to add it on android as well